### PR TITLE
dash/dg2-docs-columns-included

### PR DESCRIPTION
**Pull request by ddragula**
Added a docs paragraph for `columns.included` and `column.enabled` options.

### CODE REVIEW

File: docs/datagrid/columns.md [hunk 1]
```diff
@@ -3,6 +3,34 @@ Column options in the Datagrid
 The DataGrid provides flexible configuration options to meet your specific needs.
 for your requirements.
```
Comment (stitot) [inline docs/datagrid/columns.md:4]: Delete this line

File: docs/datagrid/columns.md [hunk 2]
```diff
@@ -3,6 +3,34 @@ Column options in the Datagrid
...
 
+## Including columns in Datagrid
```
Comment (stitot) [inline docs/datagrid/columns.md:6]:
```suggestion
## Including columns in DataGrid
```

File: docs/datagrid/columns.md [hunk 3]
```diff
@@ -3,6 +3,34 @@ Column options in the Datagrid
...
+By default, all columns from the DataTable are imported into the DataGrid in the order they are declared.
+This can be modified using the following options:
```
Comment (stitot) [inline docs/datagrid/columns.md:8]:
```suggestion
This can be modified using one of the following options:
```

File: docs/datagrid/columns.md [hunk 4]
```diff
@@ -3,6 +3,34 @@ Column options in the Datagrid
...
+- [`settings.columns.included`](https://api.highcharts.com/dashboards/#interfaces/DataGrid_Options.ColumnsSettings#included): An array of column IDs specifying which columns from DataTable to display and in what order. If `undefined`, all columns are shown in their original order.
+- [`column.enabled`](https://api.highcharts.com/dashboards/#interfaces/DataGrid_Options.IndividualColumnOptions#enabled): A boolean option controlling whether a specific column should be rendered. Defaults to `true`; set to `false` to exclude the column from the Datagrid.
```
Comment (stitot) [inline docs/datagrid/columns.md]:
```suggestion
- [`columns[columnId].enabled`](https://api.highcharts.com/dashboards/#interfaces/DataGrid_Options.IndividualColumnOptions#enabled): A boolean option controlling whether a specific column should be rendered. Defaults to `true`; set to `false` to exclude the column from the Datagrid.
```

**Review by karolkolodziej — APPROVED** [review]

File: docs/datagrid/columns.md (hunk 4 above)
Comment (stitot) [reply docs/datagrid/columns.md]: Should we also be specific on that this option only affects columns added to settings.columns.included? You can't enable a column that is not added to settings.columns.included (if settings.columns.included is defined)

**Review by stitot — CHANGES_REQUESTED** [review]

**Review by stitot — APPROVED** [review]

File: docs/datagrid/columns.md (hunk 4 above)
Comment (ddragula) [reply docs/datagrid/columns.md]: I was thinking about: `If 'settings.columns.included' is defined, you can not enable a column that is not added there.`, but I don't know if it won't cause more confusion than it solves.

Comment (stitot) [reply docs/datagrid/columns.md]: Hm...... What about something along the lines of (quite verbose, but I think we need to be): 

`settings.columns.included` defines which columns from the DataTable that should be imported to the DataGrid in what order. Can be used to exclude unwanted columns and/or change the order of columns.

The `columns` root option can then be used to further configure individual columns defined in `settings.columns.included`, and `columns[columnId].enabled` can be set to false to exclude that particular column.

Note that `settings.columns.included` is optional, and if not set the `columns` root option can be used to configure all columns from DataTable, and not just the ones defined in `settings.columns.included`.

Comment (stitot) [reply docs/datagrid/columns.md]: Or, ChatGPTs suggestion when asked to make that that text shorter and more to the point (should use ChatGPT much more for this!):

`settings.columns.included` specifies which DataTable columns to import into the DataGrid and their order. It allows for excluding or reordering columns.

The `columns` root option can further configure these selected columns, and `columns[columnId].enabled` can exclude specific columns.

If `settings.columns.included` is not set, the `columns` root option can configure all DataTable columns.

**Pull request merged**